translations/en.json

```diff
@@ -291,13 +291,13 @@
                 "popover": {
                     "role": {
                         "editor": "Editor",
-                        "editorDescription": "It will be able to view and edit the file or folder",
+                        "editorDescription": "It will be able to view and edit the item or folder",
                         "viewer": "Viewer",
-                        "viewerDescription": "It will only be able to view or download the file or folder"
+                        "viewerDescription": "It will only be able to view or download the item or folder"
```

Comment:
I think we should consider removing _or folder_. I interpret _item_ as both a file or a folder.